third_party/github.com/hashicorp/terraform-provider-google-beta/google-beta/provider/provider.go

```diff
@@ -2035,6 +2034,12 @@ func ProviderConfigure(ctx context.Context, d *schema.ResourceData, p *schema.Pr
 		}
 	}
 
+	err = transport_tpg.SetEndpointDefaults(d)
```

Comment:
I assume this is upstream code?  We would normally try to keep err local, e.g.

```
if err := transport...; err != nil {
```

Reply:
https://github.com/GoogleCloudPlatform/magic-modules/pull/9463/files#diff-4f9f71ccc5b601b48f04a9252d6c74e40b081c78c30265c1cc0277e29f689ed8R757 I'm ok patching this but this is C&P from the listed PR.